src/main/java/seedu/address/model/person/Github.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.model.person;
+
+/**
+ * Represents a person's Github details in JABPro.
+ */
+public class Github {
+
+    public final String value;
+
+    /**
+     * Constructs a {@code Github}.
+     * @param github A valid Github username
+     */
+    public Github(String github) {
+        value = github;
```

Comment:
`requireNonNull(address); `
` checkArgument(isValidAddress(address), MESSAGE_CONSTRAINTS);` - adapt accordingly, this is from address
        
   Need to have requireNonNull, and checkArgument. Might want to have a initial value as a default